agent/agent.go

```diff
@@ -276,11 +276,9 @@ func (a *Agent) fetchProjectConfig(ctx context.Context, tc *taskContext) error {
 	if err != nil {
 		return errors.Wrap(err, "error getting version")
 	}
-	project := &model.Project{}
-	// TODO: populated config from parser project
-	// later will want a separate communicator route
-	if _, err = model.LoadProjectInto([]byte(v.Config), v.Identifier, project); err != nil {
-		return errors.Wrapf(err, "error reading project config")
+	p, err := a.comm.GetProject(ctx, tc.task)
```

Comment:
I don't entirely follow the change. Is `a.comm.GetVersion` still sending the config as yaml? Does there need to be a second PR to remove that?